contributors/devel/godep.md

```diff
@@ -90,6 +90,7 @@ rm -rf vendor
 ./hack/update-bazel.sh
 ./hack/update-godep-licenses.sh
 ./hack/update-staging-client-go.sh
+./hack/update-staging-godeps.sh
```

Comment:
Ah, i didn't read the entire doc. As you said, if one follows the doc and creates a dedicated GOPATH, then it's not a problem.

Maybe the comment could say "If you haven't followed this doc step-by-step and haven't created a dedicated GOPATH, make sure there is no client-go or other staging repo in $GOPATH before running the next command."